File: Assets/Scripts/ScrollUVClose.js
#pragma strict

function Update () {
	var mat = this.renderer.material;
	var offset : Vector2 = mat.mainTextureOffset;
	
	
	offset.x = transform.position.x / 200;
	offset.y = transform.position.y / 200;
	
	mat.mainTextureOffset = -(offset);
}

File: Assets/Scripts/ScrollUVClose.cs
using UnityEngine;
using System.Collections;

public class ScrollUVClose : MonoBehaviour {

	// Update is called once per frame
	void Update () {
		Material mat = this.renderer.material;
		Vector2 offset = mat.mainTextureOffset;

		offset.x = transform.position.x / 200;
		offset.y = transform.position.y / 200;
		
		mat.mainTextureOffset = -(offset);
	}
}
